pkg/api/install/install.go

```diff
@@ -408,6 +426,17 @@ func init() {
 				return true, authorizationv1.Convert_api_RoleBinding_To_v1_RoleBinding(a, b, s)
 			}
 
+		case *authorizationv1.IsPersonalSubjectAccessReview:
```

Comment:
Right now these are all unified, will eventually generate them.  Rather
have them in one place for now.

On Thu, Jun 23, 2016 at 9:07 AM, David Eads <notifications@github.com>
wrote:

> In pkg/api/install/install.go
> <https://github.com/openshift/origin/pull/9498#discussion_r68230313>:
>
> > @@ -408,6 +426,17 @@ func init() {
> >  				return true, authorizationv1.Convert_api_RoleBinding_To_v1_RoleBinding(a, b, s)
> >  			}
> >
> > +		case *authorizationv1.IsPersonalSubjectAccessReview:
>
> Can we keep this in the authorization package?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9498/files/a068e55aa25db3fc149aa1d896426365a7734df5#r68230313>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p5_DuqzvtktU9KKWx488ydjDGIp_ks5qOoULgaJpZM4I8Zon>
> .
>